pkg/analysis/utils/testdata/src/b/structs.go

```diff
@@ -61,3 +63,10 @@ type StructWithOneNonOmittedFieldAndMinProperties struct {
 	// +optional
 	Int int32 `json:"int,omitempty"` // want "zero value is valid" "validation is not complete"
 }
+
+// Struct with an omitted required field.
+// The zero value of the struct is `{}` which is not valid because it does not satisfy the required marker on the string field.
+type StructWithOmittedRequiredField struct {
+	// +required
+	String string `json:"string,omitempty"` // want "zero value is valid" "validation is not complete"
```

Comment:
Is this the correct message here?

The `zero value is valid` is throwing me off a bit because in this case isn't the zero valid inherently invalid?